code/modules/awaymissions/mission_code/Academy.dm

```diff
@@ -167,6 +167,9 @@
 	var/used = 0
 	var/rigged = -1
 
+/obj/item/weapon/dice/d20/fate/one_use
+	reusable = 0

```

Comment:
it's a child item (AKA an item that inherits all of the parent item's traits, and can be tweaked without altering the parent item) that is one-use